tests/update/updates_packagekit_gpk.pm

```diff
@@ -45,6 +45,11 @@ sub run {
     while (1) {
         x11_start_program("gpk-update-viewer");
 
+        if ($testapi::username eq 'root') {
+            assert_screen 'package-updater-privileged-user-warning', 60;
+            send_key "alt-a" if (match_has_tag("package-updater-privileged-user-warning"));
```

Comment:
that does not seem to make sense. the assert_screen has only one tag and here you check for that tag. When do you expect the send-key *not* to be called?